v2/robotmk/src/config/external.rs

```diff
@@ -49,88 +48,28 @@ pub struct ExecutionConfig {
     pub timeout: u64,
 }
 
-#[derive(Clone, Deserialize)]
+#[derive(Deserialize)]
 #[serde(tag = "type")]
-#[cfg_attr(test, derive(Debug, PartialEq))]
 pub enum EnvironmentConfig {
     System,
     Rcc(RCCEnvironmentConfig),
 }
 
-#[derive(Clone, Deserialize)]
-#[cfg_attr(test, derive(Debug, PartialEq))]
+#[derive(Deserialize)]
 pub struct RCCEnvironmentConfig {
     pub binary_path: Utf8PathBuf,
     pub robot_yaml_path: Utf8PathBuf,
     pub build_timeout: u64,
 }
 
-#[derive(Clone, Deserialize)]
```

Comment:
FYI: Robert mentioned that he considers leaving some nice to have traits as good practice, albeit this is of course completely up the team (except maybe if binary size is a concern).

Reply:
Ok, thanks for asking. Maybe we should review this at some point and decide on a standard set of traits which we always derive if possible.